### ISSUE TITLE
senderAddress should automatically convert hex format to base58

### ISSUE THREAD

**Issue by hayeah** [qtumproject/qtumjs#2]
The qtumd RPC does not support senderAddress in hexadecimal format. This makes the qtumjs API rather awkward when senderAddress is involved.

Should detect hex address and convert that automatically to base58 format.

**Comment by pplam** [comment]
@hayeah hey man, did you fix the bug? Can it deal with overloaded methods now?

**Comment by hayeah** [comment]
the qtumd core team is adding hex address support, I am waiting on that.

what do you mean by overloaded methods?

**Comment by pplam** [comment]
Suppose I have two methods share the same name in my abi, i.e. `transfer(address,amount)` and `tansfer(address,amount,data)`, your `qtumjs` doesn't work. I have figured it out in a issue before, but  somehow it was deleted. Please have a look at here:
https://github.com/qtumproject/qtumjs/blob/dea61ddb4b29a7ff42a64d0378e28227bf38f985/src/Contract.ts#L195-L204

**Comment by pplam** [comment]
And another issue, I don't know why some events are decoded to `null`, so I have to use `ethjs-abi` to decode them manually

**Comment by hayeah** [comment]
can you give me the Solidity example where the events are decoded to null? 

I'll debug it. I've created an issue for this problem: https://github.com/qtumproject/qtumjs/issues/4

**Comment by hayeah** [comment]
@pplam method overloading support is added. see:

https://qtumproject.github.io/qtumjs-doc/#method-overloading

i'll close this issue in favour of #4
